lib/decidim/vocdoni/component.rb

```diff
@@ -6,6 +6,7 @@
   component.engine = Decidim::Vocdoni::Engine
   component.admin_engine = Decidim::Vocdoni::AdminEngine
   component.icon = "media/images/decidim_vocdoni.svg"
+  component.stylesheet = "decidim/vocdoni/vocdoni"
```

Comment:
This introduces the stylesheet in every page along Decidim. Maybe we should make sure to namespace any potential CSS class that can have collisions with other with the same name in places unrelated. For instance, I'm thinking on the ".accordion" class.

Reply:
Oh I see what you mean. I missed that.

Done at aad92441ee9fb2fe024bb242da7f8634655dc387